addon/components/o-s-s/search-field.ts

```diff
@@ -0,0 +1,33 @@
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+import Component from '@glimmer/component';
+import type IntlService from 'ember-intl/services/intl';
+
+interface OSSSearchFieldArgs {
+  value: string;
+  placeholder?: string;
+  // eslint-disable-next-line no-unused-vars
+  onChange: (value: string) => void;
+}
+
+export default class OSSSearchFieldComponent extends Component<OSSSearchFieldArgs> {
+  @service declare intl: IntlService;
+  @tracked value = this.args.value;
+
+  get placeholder(): string {
+    return this.args.placeholder || this.intl.t('oss-components.access-panel.search_placeholder');
```

Comment:
nit: even if the translation contains the same text as `access-panel.search_placeholder`, i think it's better to still create a `oss-components.search_field.placeholder` translation here.

This way, if someday someone refactors or removes the `access-panel` component, they don't accidentally cause a side effect "Missing translation" here.